### PR TITLE
Mixer URLFollow

### PR DESCRIPTION
**Pull request by MasterGunner**
A URL Follow for Mixer streams, based off the Twitch module.

Should be able to generate the client-id from here: https://mixer.com/lab/keypopup

### CODE REVIEW

File: desertbot/modules/urlfollow/Mixer.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+            return
+        channel = match.group('mixerChannel')
+
+        if self.mixerClientID is None:
+            return '[Mixer Client ID not found]'
+
+        chanData = {}
+        channelOnline = False
+        mixerHeaders = {'Accept': 'application/json',
+                         'Client-ID': self.mixerClientID}
```
Comment (Heufneutje) [inline desertbot/modules/urlfollow/Mixer.py]: Indentation is a little off here. Looks good otherwise.

Comment (MasterGunner) [reply desertbot/modules/urlfollow/Mixer.py]: Indentation is identical to the Twitch module, but I'm no pythonsm'n and could be missing something.

Comment (Heufneutje) [reply desertbot/modules/urlfollow/Mixer.py]: You have a space too many it seems.

**Comment by Heufneutje** [comment]
Should be fine to merge now.

**Comment by StarlitGhost** [comment]
API key generated and added. It says a current stream from the mixer frontpage is offline, though.
Could just be a delay on the API updating, like Twitch has sometimes.

**Comment by StarlitGhost** [comment]
I tested with more streams and all of them say they are offline.
In particular, https://mixer.com/Monstercat should always be live so I think rules out API lag.

**Pull request merged**